lib/load_disposable_domain.rb

```diff
@@ -0,0 +1,7 @@
+class LoadDisposableDomain
+  def self.load_disposable_domains(url)
+    Faraday.get(url).body.each_line do |line|
+      DisposableDomain.find_or_create_by(name: line)
```

Comment:
I'm pretty sure this one-at-a-time will be much slower than a bulk insert, have we looked into the [INSERT ... ON CONFLICT](https://www.postgresql.org/docs/current/sql-insert.html#SQL-ON-CONFLICT) approach for a bulk load?

Reply:
[insert_all](https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-insert_all) is probably the preferable approach for this